modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/WFSDataStoreFactory.java

```diff
@@ -129,7 +129,19 @@ public WFSDataStore createDataStore(final Map<String, Serializable> params)
 
         return dataStore;
     }
-    
+
+    public HTTPClient getHttpClient(final Map<String, Serializable> params) throws IOException {
```

Comment:
Public methods should have javadocs (yes, I know there are many that still do not)